docs/static/releasenotes.asciidoc

```diff
@@ -1823,4 +1847,4 @@ Here are the plugin changes.
   - Tweaked logging statements to reduce verbosity
   - Fixed numerous issues relating to builds on Travis https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/799[#799]
 * logstash-output-s3
-  - Fixed issue where on restart, 0 byte files could erroneously be uploaded to s3 https://github.com/logstash-plugins/logstash-output-s3/issues/195[#195]
+  - Fixed issue where on restart, 0 byte files could erroneously be uploaded to s3 https://github.com/logstash-plugins/logstash-output-s3/issues/195[#195]
```

Comment:
Why this changed? CR LF chars perhaps

Reply:
Possibly. Wasn't obvious to me either...